src/Components/PelcroModalController/PelcroModalController.service.js

```diff
@@ -79,11 +80,13 @@ export const initPaywalls = () => {
 };
 
 export const loadPaymentSDKs = () => {
-  // Load stripe's SDK
-  window.Pelcro.helpers.loadSDK(
-    "https://js.stripe.com/v3/",
-    "pelcro-sdk-stripe-id"
-  );
+  // Lazy load stripe's SDK
+  const { whenUserReady } = usePelcro.getStore();
+  whenUserReady(() => {
+    if (!window.Stripe) {
```

Comment:
Stripe's SDK does this check internally, so it's redundant here, but that part is not well documented on their side, so I kept it here to make things clear for people reading the code